pkg/cmd/util.go

```diff
@@ -24,6 +24,8 @@ func AddCommonFlags(c *cobra.Command, cfg *api.Config) {
 		"DEPRECATED: Always pull the builder image even if it is present locally")
 	c.Flags().VarP(&(cfg.BuilderPullPolicy), "pull-policy", "p",
 		"Specify when to pull the builder image (always, never or if-not-present)")
+	c.Flags().Var(&(cfg.PreviousImagePullPolicy), "incremental-pull-policy",
+		"Specify when to pull the previous image for incremental builds (always, never or if-not-present)")
```

Comment:
Unfortunately there's a slight problem with this, and main reason I haven't submitted my changes yet. Since @mfojtik created [this String function](https://github.com/openshift/source-to-image/blob/master/pkg/api/types.go#L319-L324) which *always* shows `if-not-present` which is not true for previous image policy. His reasoning at the time was to show that default, but we can't do it correctly. We need to remove that string function and place the default information inside the description. I'm also looking at other option of using `StringVarP` method and transform the string into the PullPolicy type.